frontend/include/chpl/framework/Context.h

```diff
@@ -157,6 +188,14 @@ class Context {
   // tracks the nesting of queries, displayed during query tracing
   int queryTraceDepth = 0;
 
+  using ErrorCollectionEntry = std::pair<std::vector<owned<ErrorBase>>*, const querydetail::QueryMapResultBase*>;
+  // If this vector is non-empty, the top element is a collection into
```

Comment:
vectors have a front and back but not a top, could you update the comment to say front or back?